additional-functions/manage-restrictions.md

```diff
@@ -0,0 +1,49 @@
+## Manage restrictions
+
+Restrictions in Mews are configured as combinations of [conditions and exceptions](https://help.mews.com/en/articles/4244406-how-restrictions-work). Below are the three [restriction types](../operations/services.md#restriction-type) that can be applied:
+ 
+* Start: Closed to arrival, meaning guests cannot check-in within the specified dates.
+* End: Closed to departure, meaning guests cannot check-out within the specified dates.
+* Stay: Closed to stay, meaning guests cannot stay overnight within the specified dates.
+
+Using the Connector API, restrictions are managed by [retrieving the existing restrictions](../operations/services.md#get-all-restrictions), [adding a new restriction](../operations/services.md#add-restrictions) or [deleting an existing one](../operations/services.md#delete-restrictions). 
+A restriction can be applied to either rates or resources of a specified reservable service. A list of all services configured at a property is requested using [Get all services](../operations/services.md#get-all-services) and the services to which restrictions can be applied will have the [service type](../operations/services.md#service-type) `Reservable`.
+
+### Adding a rate restriction
+
+Properties can [configure their Mews rates](https://help.mews.com/en/articles/4244388-create-a-rate) as individual ones, derived from another rate, as well as a part of a rate group. Integration partners use [Get all rates](../operations/services.md#get-all-rates) to retrieve the rates configured at the property and the unique identifiers required for applying restrictions. To request information about rate groups, include the relevant [rate extent](../operations/services.md#rate-extent).
```

Comment:
I'm not a fan of linking external stuff from here. Not sure the value outweights the maintenance costs.